website/docs/configuration/resources.html.md

```diff
@@ -92,6 +92,8 @@ There are **meta-parameters** available to all resources:
         which will match all attribute names. Using a partial string together
         with a wildcard (e.g. `"rout*"`) is **not** supported.
 
+  -> **Note**: Interpolations are **not** supported in `lifecycle` yet (see [issue #3116](https://github.com/hashicorp/terraform/issues/3116)).
```

Comment:
I think having this in a highlighted callout is sufficient and matches the other listed caveats, so we don't need the extra  emphasis. We also don't generally have GH links in the docs themselves. For consistency, can we change this to:

```
    -> Interpolations are not currently supported in the `lifecycle` configuration block.
```

Reply:
Formatting is fine.
I like references to issues while reading through the docs as they offer context and explanation. Like the [GH reference in `aws_iam_instance_profile` doc](https://github.com/terraform-providers/terraform-provider-aws/blob/master/website/docs/r/iam_instance_profile.html.markdown#argument-reference).

If you really disallow GH links, can we at least add it as a HTML comment, please?

Reply:
Having a Github issue reference in doc could also minimize duplicate issues. Just saw that you resolved the [7th](https://github.com/hashicorp/terraform/search?q=3116&type=Issues) [duplicate](https://github.com/hashicorp/terraform/issues/17775) for the "lifecycle interpolation" issue yesterday.

Reply:
Fair enough. Let's update the text and leave the link to the discussion.

Reply:
Updated it. Can you squash the two commits when merging, please?

Reply:
👍 thanks!